feature/experimental/isis/ate_tests/isis_drain_test/README.md

```diff
@@ -0,0 +1,27 @@
+# RT-2.11: IS-IS Drain Test
```

Comment:
How did you pick this test#?

Please use RT-2.12 instead. I will update the proto in Google